lib/ios_deep_link_manager.dart

```diff
@@ -3,7 +3,7 @@ import 'package:flutter/services.dart';
 
 class IOSDeepLinkManager extends DeepLinkNavigator {
   static final IOSDeepLinkManager _instance = IOSDeepLinkManager._internal();
-  static const MethodChannel _platform = MethodChannel('com.tin.mobile');
+  static const MethodChannel _platform = MethodChannel('com.ensemble.platform');
```

Comment:
com.ensembleui - that's our domain (not com.ensemble)

Reply:
I already updated it to ```com.ensembleui.host.platform```

Reply:
Added in the starter project. 
PR - https://github.com/EnsembleUI/starter/pull/33